shopinvader_quotation_payment/services/cart.py

```diff
@@ -13,5 +13,5 @@ class CartService(Component):
     def _get_available_payment_mode(self, cart):
         for line in cart.order_line:
             if line.product_id.only_quotation:
-                return []
+                return self.env["shopinvader.payment"].browse(False)
```

Comment:
Can you add a comment describing functionally the purpose of this code.

Reply:
done